tests/resume/accounts.pact

```diff
@@ -0,0 +1,235 @@
+;; accounts module, admin keyset, and table
```

Comment:
This needs to be stripped down to exactly what you're regressing, or instead use `examples/accounts`